pages/home.jsx

```diff
@@ -101,13 +105,15 @@ var BlogSection = React.createClass({
     }
   },
   componentDidMount: function() {
-    var self = this;
-    blogFeedLoader(this, window, "https://blog.webmaker.org/tag/teachtheweb/feed", function(data) {
-      self.setState({
-        featuredPostData: data.featuredPostData,
-        latestPostsData: data.latestPostsData
+    this.props.loadBlogPosts(function(data) {
```

Comment:
I decided to remove even the feed URL from here, since right now we don't even need to pass that in since we're not planning on retrieving blog posts from multiple sources or anything anytime soon...